secrets/helper/helper.go

```diff
@@ -139,6 +143,39 @@ func InitNetworkingPrivateKey(secretsManager secrets.SecretsManager) (libp2pCryp
 	return libp2pKey, keyErr
 }
 
+func InitValidatorBLSSignature(
+	secretsManager secrets.SecretsManager, account *wallet.Account, chainID int64) ([]byte, error) {
+	if secretsManager.HasSecret(secrets.ValidatorBLSSignature) {
+		return nil, fmt.Errorf(`secrets "%s" has been already initialized`, secrets.ValidatorBLSSignature)
+	}
+
+	// Generate the signature
+	s, err := MakeKOSKSignature(
+		account.Bls,
+		types.Address(account.Ecdsa.Address()),
+		chainID,
+		bls.DomainValidatorSet,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	sb, err := s.Marshal()
+	if err != nil {
+		return nil, err
+	}
+
+	// Write the signature to the secrets manager storage
+	if setErr := secretsManager.SetSecret(
+		secrets.ValidatorBLSSignature,
+		[]byte(hex.EncodeToString(sb)),
```

Comment:
Can't we just same `sb` instead of hex encoding it to string and then converting back to bytes?

Reply:
I think secret manager can not handle any byte slice, i think it should be human readable string (converted to bytes)

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1233/commits/042fb94759287b200e2a68211f691410f64b191d

Reply:
Reverted, I forgot that I tried it and the marshaling is binary so it ends up garbled chars in json, we need to encode to string then pass bytes.